adapters/mqtt-vertx/src/main/java/org/eclipse/hono/adapter/mqtt/Config.java

```diff
@@ -29,6 +32,10 @@
 @Configuration
 public class Config extends AdapterConfig {
 
+    @Autowired(required=false)
```

Comment:
please also change the copyright header to include the current year:
```
Copyright (c) 2016, 2017 Red Hat and others.
```
You may also want to add `Bosch Software Innovations` to the `Contributors`.